nifi-fn/.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
I think traditionally we just inherit the root `.gitignore` file.

Reply:
Makes sense. removed.